survey/src/main/java/com.quickbirdstudios.surveykit/backend/views/question_parts/DateTimePickerPart.kt

```diff
@@ -0,0 +1,130 @@
+package com.quickbirdstudios.surveykit.backend.views.question_parts
+
+import android.app.DatePickerDialog
+import android.app.TimePickerDialog
+import android.content.Context
+import android.os.Parcelable
+import android.util.AttributeSet
+import android.view.Gravity
+import android.widget.*
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.core.content.ContextCompat
+import com.quickbirdstudios.surveykit.R
+import com.quickbirdstudios.surveykit.SurveyTheme
+import com.quickbirdstudios.surveykit.backend.views.main_parts.StyleablePart
+import kotlinx.android.parcel.Parcelize
+import java.util.*
+
+internal class DateTimePickerPart @JvmOverloads constructor(
```

Comment:
Providing a default value does not work.

A good way to extract/set the date/time from the `DateTimePickerPart` has to be found.

I'd also like the `DateTimePickerPart` to be more intuitive:
- clicking on the date opens the date picker
- clicking on the time opes the time picker
- the date/time display should be customizable

All of these are improvement suggestions for another iteration.

Reply:
Yes, the date/time definition and setting could be improved, I did it like this to be flexible with any date API the client would use (Date, Calendar, Joda) but it's a bit redundant, do you have any suggestions?

Reply:
Id leave it as it is.

However a test with a default value working would be really nice!